ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigOptions.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { ReactElement } from 'react';
+import { FormikProps } from 'formik';
+import {
+    Divider,
+    Flex,
+    FlexItem,
+    Form,
+    PageSection,
+    Stack,
+    StackItem,
+    TextArea,
+    TextInput,
+    TimePicker,
+    Title,
+} from '@patternfly/react-core';
+
+import DayPickerDropdown from 'Components/PatternFly/DayPickerDropdown';
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import RepeatScheduleDropdown from 'Components/PatternFly/RepeatScheduleDropdown';
+import { getTimeHoursMinutes } from 'utils/dateUtils';
+
+import { ScanConfigFormValues } from './useFormikScanConfig';
+
+export type ScanConfigOptionsProps = {
+    formik: FormikProps<ScanConfigFormValues>;
+};
+
+function ScanConfigOptions({ formik }: ScanConfigOptionsProps): ReactElement {
+    function handleSelectChange(id: string, value: string): void {
+        formik.setFieldValue('parameters.daysOfWeek', []);
+        formik.setFieldValue('parameters.daysOfMonth', []);
+        formik.setFieldValue(id, value);
+    }
+
+    function handleTimeChange(
+        _event: React.FormEvent<HTMLInputElement>,
+        time: string,
+        hour?: number,
+        minute?: number,
+        _seconds?: number,
+        isValid?: boolean
+    ): void {
+        formik.setFieldTouched('parameters.time', true, true);
```

Comment:
We haven't had to use the `setFieldTouched` method in this app before; instead, we've used `onBlur` on the field. Are we using it here because the PF component doesn't support onBlur, or to get instance validation of the time format?

Reply:
Huh I didn't think onBlur was supported looking at the [docs](https://v4-archive.patternfly.org/v4/components/time-picker#props) and typescript complains when onBlur is used. Yet it appears to work as you would expect when testing it out and I see 'onBlur' in the [component on github](https://github.com/patternfly/patternfly-react/blob/a816ddd2e919197e916f05806bd7f93e208a070b/packages/react-core/src/components/TimePicker/TimePicker.tsx#L27). I'll reach out to the patternfly team.

Reply:
I was able to get rid of `formik.setFieldTouched` on both... 

- `TimePicker` using `inputProps`
- `DayPickerDropdown` component using the `Select` -> `toggleId` prop